ext/sst/gem5.hh

```diff
@@ -80,7 +80,6 @@
 #include <sst/core/component.h>
 
 #include <sst/core/interfaces/stringEvent.h>
-#include <sst/core/interfaces/simpleMem.h>
```

Comment:
Why this change?

Reply:
My mistake here. This change was supposed to be in https://github.com/gem5/gem5/pull/396 to remove all deprecation warning messages. Do I revert this and create a different PR for this change? Or do I update the commit note?

Reply:
You don't have to make a new PR, but if you could split this out into its on commit it would be better.

Reply:
I have split this into two commits in the latest patch.